src/sentry/static/sentry/app/views/eventsV2/table/index.tsx

```diff
@@ -79,6 +79,7 @@ class Table extends React.PureComponent<TableProps, TableState> {
     const url = `/organizations/${organization.slug}/eventsv2/`;
     const tableFetchID = Symbol('tableFetchID');
     const apiPayload = eventView.getEventsAPIPayload(location);
+    setError('');
```

Comment:
@wmak Actually, can we make this as `undefined`? This implies the type should now be `string | undefined` rather than just `string`.